cfme/cloud/instance.py

```diff
@@ -240,12 +240,6 @@ def create(self, email=None, first_name=None, last_name=None, cloud_network=None
             flash.assert_success_message(
                 "VM Provision Request was Submitted, you will be notified when your VMs are ready")
 
-        row_description = 'Provision from [{}] to [{}]'.format(self.template_name, self.name)
```

Comment:
assert moved to test module and now it's soft_assert based on does_vm_exist_on_provider